packages/is-valid-prop/scripts/prepare.ts

```diff
@@ -30,18 +30,16 @@ writeFileSync(
 
   const properties = new Map(allCssProperties.map((prop) => [prop, true]))
 
-  function memo<T>(fn: (value: string) => T): (value: string) => T {
+  function memoize<T>(fn: (value: string) => T): (value: string) => T {
     const cache = Object.create(null)
     return (arg: string) => {
       if (cache[arg] === undefined) cache[arg] = fn(arg)
       return cache[arg]
     }
   }
 
-  const selectorRegex = /&|@/
```

Comment:
re-using a single Regex instance is actually better for perf reasons
otherwise it would be recreated on each `isCssProperty` call, and since it's being called quite a lot, it matters